pkgs/applications/networking/browsers/tor-browser-bundle-bin/default.nix

```diff
@@ -51,6 +51,16 @@
 # Whether to disable multiprocess support
 , disableContentSandbox ? false
 
+# Updater
+, writeShellScript
+, curl
+, gnugrep
+, gnused
+, gnupg
+, nix
+, common-updater-scripts
```

Comment:
Maybe I overlook something, but now that you use `callPackage`, are these still needed?

```suggestion
```

Reply:
Of course, you're absolutely right. I didn't really understand that part about `callPackage`, even though I've used it like that before 😅.

Reply:
Btw, did you see my comment here? https://github.com/NixOS/nixpkgs/pull/231708#issuecomment-1546856364